tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-card.html

```diff
@@ -0,0 +1,83 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../paper-spinner/paper-spinner-lite.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-card-heading/tf-card-heading.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../tf-imports/lodash.html">
+<link rel="import" href="../vz-line-chart/vz-line-chart.html">
+
+<!--
+  Renders precision-recall curves.
+-->
+<dom-module id="tf-pr-curve-card">
```

Comment:
Be aware: `tf-scalar-chart` has a lot of logic dedicated to improving performance and preventing errors in Plottable (#163, #189, #204). By creating a new component, you lose those benefits. This may be an okay first step; we can try to reconcile the hierarchy later.